WORKSPACE

```diff
@@ -1,5 +1,11 @@
 workspace(name = "org_tensorflow")
 
+load("//tensorflow:workspace.bzl", "check_version", "tf_workspace")
+
+# DO NOT ADD ANYTHING ABOVE THIS LINE!!!
```

Comment:
Done. Made the change as a separate commit but I can squash all after approval, if you like.